addons/medical_gui/functions/fnc_onKeyDown.sqf

```diff
@@ -81,4 +87,8 @@ switch (true) do {
     };
 };
 
+if (_return) then {
+    playSoundUI ["\a3\ui_f\data\sound\rscbuttonmenu\soundClick.wss", 0.09, 1];
```

Comment:
Volume will be scaled off UI volume slider and thus might not match clicking the buttons themselves. Is there a CfgSounds entry for this? (or does playSound support filenames?)

Reply:
Some BI utility script use `playSound ["SoundClick", true];`